pkg/controller/lifecycle/actuator_test.go

```diff
@@ -532,19 +537,13 @@ spec:
   selector:
     matchLabels:
       app.kubernetes.io/name: lakom
-      app.kubernetes.io/part-of: shoot-lakom-service
-`
-	if withUnhealthyPodEvictionPolicy {
-		out += `  unhealthyPodEvictionPolicy: AlwaysAllow
-`
-	}
-	out += `status:
+      app.kubernetes.io/part-of: shoot-lakom-service` + unhealthyPodEvictionPolicyStr + `
+status:
```

Comment:
I think that the nicer formatting will be to set the newline here and not in the variable.

```suggestion
      app.kubernetes.io/part-of: shoot-lakom-service
  ` + unhealthyPodEvictionPolicyStr + `
status:
```

Reply:
When `withUnhealthyPodEvictionPolicy == false`, there will be one unneeded empty line just before status and the test will be failing.

The alternative is

```go
func expectedSeedPDB(namespace string, withUnhealthyPodEvictionPolicy bool) string {
	unhealthyPodEvictionPolicyStr := ""
	if withUnhealthyPodEvictionPolicy {
		unhealthyPodEvictionPolicyStr = `  unhealthyPodEvictionPolicy: AlwaysAllow
`
	}
	return `apiVersion: policy/v1
kind: PodDisruptionBudget
metadata:
  creationTimestamp: null
  labels:
    app.kubernetes.io/name: lakom
    app.kubernetes.io/part-of: shoot-lakom-service
  name: extension-shoot-lakom-service
  namespace: ` + namespace + `
spec:
  maxUnavailable: 1
  selector:
    matchLabels:
      app.kubernetes.io/name: lakom
      app.kubernetes.io/part-of: shoot-lakom-service
` + unhealthyPodEvictionPolicyStr + `status:
  currentHealthy: 0
  desiredHealthy: 0
  disruptionsAllowed: 0
  expectedPods: 0
`
}
```

But now `status` is at the end of a line and `unhealthyPodEvictionPolicyStr` have a trailing new line. 
Both options have some prons and cons with regards to formatting, none of them looks cool 😎

Reply:
Ah, I see. Both ways are not pleasant to the eye.